### PR TITLE
Remove ASCII Header.

### PR DESCRIPTION
**Pull request by TomaNikolov**

### CODE REVIEW

File: cli/trigger.go [hunk 1]
```diff
@@ -7,7 +7,6 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/bitrise-io/bitrise/models"
-	"github.com/bitrise-io/bitrise/version"
```
Comment (rosen-vladimirov) [inline cli/trigger.go:10]: why is this line removed, but all others are commented?

**Review by rosen-vladimirov — APPROVED** [review]

File: cli/trigger.go (hunk 1 above)
Comment (TomaNikolov) [reply cli/trigger.go:10]: Version is only used in `PrintBitriseHeaderASCIIArt(version.VERSION)`. Since this method is commented the import is not needed anymore, However, if we uncomment the method, the VScode Go plugin most likely will add this line automatically.
